main.py

```diff
@@ -103,18 +86,28 @@ def create_mw(self, request: Response) -> JSONResponse:
             raise HTTPException(
                 400, detail='Setting a maintenance status is not allowed'
             )
+        # if 'id' in data:
+        #     raise HTTPException(
+        #         400, detail='Setting a maintenance id is not allowed'
+        #     )
         try:
-            if data.get('id') == '':
-                del data['id']
             maintenance = MW.parse_obj(data)
             force = data.get('force', False)
-        except ValidationError as err:
-            raise HTTPException(400,
-                                detail=f'{err.errors()[0]["msg"]}') from err
-        try:
+            if not force:
```

Comment:
Wasn't `force` only for bypassing the overlap start/end check? `force` to bypass start/end check as long as network operators are OK with it is useful for certain cases, but now in this case here, it seems that it'd be encouraging sloppy payloads, which also would result in a [`log.error` on topology](https://github.com/kytos-ng/topology/blob/master/main.py#L1195-L1199), which wouldn't cause issues, but it's an error message that a network operator might find surprising (although if they're using `force` they should know what they're really doing), which brings me to the point, are we not stretching much what `force` was originally for? Also it's worth documenting what `force` is meant to be used for in `openapi.yml`. Let me know if you had any other use case in mind, or see cases where indeed it'd be positive to use `force` here. Currently, we don't support hard deletions on topology for items.